### ISSUE TITLE
nnenum not working

### ISSUE THREAD

**Issue by cherrywoods** [dlshriver/DNNV#98]
Hi,
When I run nnenum in dnnv it gives me an error message that is suggestive of a to new numpy version. Is this due to nnenum or should it be managed in dnnv?

```
.../python3.9/site-packages/numpy/lib/npyio.py:528: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray.
  arr = np.asanyarray(arr)
dnnv.verifiers.nnenum
  result: NnenumError(Return code: 1)
  time: 0.6607
```

BaB is running fine. 
I did:
```bash
python3 -m virtualenv venv --python python3.9
source venv/bin/activate
pip install dnnv
dnnv_manage install bab nnenum
git clone https://github.com/dlshriver/dnnv-benchmarks benchmarks
dnnv benchmarks/benchmarks/ACAS_Xu/properties/property_2.py -N "N" benchmarks/benchmarks/ACAS_Xu/onnx/N_2_1.onnx --save-violation "CX.npy" --nnenum
```

Kind regards,
David

**Comment by dlshriver** [comment]
Thank you for your interest in DNNV. You are correct that this is a bug. An error in DNNV's installation of nnenum results in a missing dependency. I've got a fix in the works, hopefully I can push it by the end of the day.

**Comment by dlshriver** [comment]
This should be fixed in commit [f7a39b1](https://github.com/dlshriver/dnnv/commit/f7a39b1df66c1dfd5485e284ad0210ed0d881d41) on the develop branch. It will be fixed in the next release, which should be within the next week or so. There's a couple more change that I want to make first.

If you need this fix now, then instead of `pip install dnnv` you can use `pip install git+https://github.com/dlshriver/dnnv.git@develop`, to get the latest changes on the develop branch.

**Comment by cherrywoods** [comment]
Great! The issue is resolved on my side as well. 
Thanks for fixing this. For me, DNNV is super useful for testing out different verifiers.
